src/content/api/compilation-hooks.md

```diff
@@ -206,68 +216,68 @@ Parameters: `chunks`
 
 Fired after chunk optimization has completed.
 
-Parameters: `chunks`
+- Callback Parameters: `chunks`
 
 
 ### `optimizeTree`
 
 `AsyncSeriesHook`
 
-Optimize the dependency tree asynchronously.
+Called before optimizing the dependency tree. A plugin can tap into this hook to perform a dependency tree optimization.
 
-Parameters: `chunks` `modules`
+- Callback Parameters: `chunks` `modules`
 
 
 ### `afterOptimizeTree`
 
 `SyncHook`
 
-...
+Fired after the dependency three optimization has completed with success.
 
-Parameters: `chunks` `modules`
+- Callback Parameters: `chunks` `modules`
 
 
 ### `optimizeChunkModulesBasic`
 
 `SyncBailHook`
 
-...
+W> This hook will be removed in v5.0.0
 
-Parameters: `chunks` `modules`
+- Callback Parameters: `chunks` `modules`
 
 
 ### `optimizeChunkModules`
 
 `SyncBailHook`
 
-...
+Called after the three optimization, at the beginning of the chunkmodules optimization. A plugin can tap into this hook to perform a chunkmodules optimization.
```

Comment:
Can you link `chunkmodules` please

Reply:
Sure, looking in the documentation I can't find `chunkmodules`, do you mean the [code splitting](https://webpack.js.org/guides/code-splitting/) section, the stats section or something else? 

Thanks a lot for the corrections!

Reply:
I mean, it doesn't read right as `chunkmodules`, looks like it would run after the tree optimization indeed, and not three. Lets try to change it to:
```suggestion
Called after the tree optimization, at the beginning of the chunk modules optimization. A plugin can tap into this hook to perform optimizations of chunk modules.
```